ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/ip-routing.md

```diff
@@ -71,7 +71,7 @@ interface Management1
 
 | VRF | Routing Enabled |
 | --- | --------------- |
-| default | false|| default | true |
+| default | true |
```

Comment:
That is incorrect as EOS configuration snapshot below has `no ip routing`. Must be `false`.

Reply:
This is triggered by having the vrf `default` defined under `vrfs`. In those cases the previous template code printed both entries. So one for the global `ip_routing` and one for the `vrfs['default'].ip_routing`. Since the config can only be one or the other, I took the variable under the `vrfs`. The config logic is probably also faulty, so printing two lines. One "ip routing" and another "no ip routing".
I will check and sort it out. The order suddenly becomes important.

Reply:
The config logic is mixed with different behaviors between `ip` and `ipv6`:
```eos
no ip routing
ip routing vrf TEST1
no ip routing vrf TEST2
no ip icmp redirect
!
ipv6 unicast-routing
ip routing ipv6 interfaces
!
ipv6 unicast-routing vrf default
ipv6 unicast-routing vrf TEST1
```

Reply:
I will revert that change from this pr, so the documentation will still print both. And then create a separate PR to fix this.

Reply:
done